collections/attachments.js

```diff
@@ -0,0 +1,58 @@
+Attachments = new FS.Collection("attachments", {
+    stores: [ 
+        new FS.Store.FileSystem("attachments", { path: "~/uploads/attachments" }) // XXX PATH ???? 
+    ]
+});
+
+// ALLOWS
+Attachments.allow({
+    insert: function(userId, doc) {
+        return allowIsBoardMember(userId, Boards.findOne(doc.boardId));
+    },
+    update: function(userId, doc) {
+        return allowIsBoardMember(userId, Boards.findOne(doc.boardId));
+    },
+    remove: function(userId, doc) {
+        return allowIsBoardMember(userId, Boards.findOne(doc.boardId));
+    },
+    download: function(userId, doc) {
+        return true;
+    },
+    fetch: ['boardId']
+});
+
+/*  
+// XXX ??? /cfs.attachments.filerecord/insert' TypeError: Cannot call method 'depend' of undefined
+Attachments.files.attachSchema(new SimpleSchema({
+    boardId: {
+        type: String
+    },
+    cardId: {
+        type: String
+    },
+    cover: {
+        type: Boolean
+    }
+}));
+*/
+
+// HOOKS 
+
+// the Meteor/Mongo.collection resides in the files property of the FS.Collection
+Attachments.files.before.insert(function(userId, doc) {
+    doc.userId = userId;
+    doc.cover = true;
```

Comment:
I think we need a better cover system. First we need to check if the cover is an image. Second, we probably want to create another store in CollectionFS with only thumbnails large images in the board view.